config/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
```

Comment:
Should then also directly have the correct permission?

Reply:
Done